src/DotLiquid/StandardFilters.cs

```diff
@@ -116,22 +116,16 @@ public static string UrlDecode(string input)
         /// <returns></returns>

```

Comment:
Update the XML summary description of this filter to more accurately describe the behavior